### DISCUSSION TITLE
What if Livewire is auto injected into the head after other livewire CDN's are included?

### DISCUSSION THREAD

**Discussion by myregistration** [livewire/livewire#8170] [1. Help]
When I have Livewire config `inject_assets' => true` and using the following CDN version of Livewire Sortable v1 for Livewire v3 inserted in a stack before the closing body tag:

```
@pushonce('scripts-append')
    <script src="https://cdn.jsdelivr.net/gh/livewire/sortable@v1.x.x/dist/livewire-sortable.js"></script>
@endpushonce
```

I get the following error:

`index.js:3 Uncaught Livewire Sortable Plugin: window.Livewire is undefined. Make sure @livewireScripts is placed above this script include`

Are Livewire's scripts also injected right before the closing body tag, but after my stack?

```
<body>
    ...
    @stack('scripts-append')
</body>
```

Is this a bug?  How can we resolve this while still using a CDN and Livewire injection?  Thanks!

**Comment by joshhanley** [comment]
@myregistration hmm I think that the sortable package might need to be updated to ensure it is wrapped in a `document.addEventListener('livewire:init', () => { ... })` so it can be include before Livewire's scripts but doesn't load until after Livewire has started processing.
